docs/flashbots-protect/gas-fee-refunds.md

```diff
@@ -44,14 +44,40 @@ For the private transaction API: The refund recipient is the signer used on the
 
 ## How to track refunds
 
-Refunds are tracked from a start date of July 8, 2024. Refunds are sent to recipients in batches, and originate from our builder address `0xdf99A0839818B3f120EBAC9B73f82B617Dc6A555`. The recipient can track the status of their refunds using the [`flashbots_getFeeRefundTotalsByRecipient`](/flashbots-auction/advanced/rpc-endpoint#flashbots_getfeerefundtotalsbyrecipient) RPC method.
+Refunds are tracked from a start date of July 8, 2024. Refunds are sent to recipients in batches, the first batch originated from our builder address `0xdf99A0839818B3f120EBAC9B73f82B617Dc6A555` while newer batches originate from [`refunds.builder-net.eth`](https://etherscan.io/address/0x62a29205f7ff00f4233d9779c210150787638e7f). The recipient can track the status of their refunds using the [`flashbots_getFeeRefundTotalsByRecipient`](/flashbots-auction/advanced/rpc-endpoint#flashbots_getfeerefundtotalsbyrecipient) RPC method.
 
 ## Distributed refunds
 
 Refunds have been sent as part of the following on-chain transactions:
 
-| Block Number | Sender                                     | Transaction Hash                                                                                                                                                 | Refunded Amount            |
```

Comment:
Yeah I think the long table is not great. Let's remove from the docs and add to dune, which we can link out to from here.